### ISSUE TITLE
Fullscreen still has margin, grey background, and never hides the playhead

### ISSUE THREAD

**Issue by wilg** [videovillage/Frame-Issues#16]
<img width="1440" alt="screen shot 2018-04-27 at 7 53 17 pm" src="https://user-images.githubusercontent.com/54353/39391200-a56e2aa2-4a54-11e8-9f85-65417b606076.png">

**Comment by gregcotten** [comment]
Yes for fullscreen mode the idea will be to fullscreen the video (taking away margins), anything dark grey goes black, mouse will disappear when not moved for a while, and playhead will appear on mouse move

**Comment by wilg** [comment]
Great ideas!!!!!!!!!!
